snapshot/pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -930,7 +930,7 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *v1.Serv
 
 		portID := loadbalancer.VipPortID
 		newOpts := []string{lbSecGroup.ID}
-		updateOpts := neutronports.UpdateOpts{SecurityGroups: &newOpts}
+		updateOpts := neutronports.UpdateOpts{SecurityGroups: newOpts}
```

Comment:
According to the following code, this should be a pointer so the original code is correct.  Is the definition in the SDK different?
https://github.com/gophercloud/gophercloud/blob/master/openstack/networking/v2/ports/requests.go#L118
	SecurityGroups      *[]string      `json:"security_groups,omitempty"`

Reply:
Yeah, not sure why the load_balancer and other non-related snapshot items were touched anyway... bad merge with something somewhere along the line maybe?

Reply:
When I ran the glide up command it pulled in a version of gophercloud where those two fields were not pointers.  I am really not sure how that happened.  glide up is voodoo.  I'll try to fix that now.